src/server/db_slice.h

```diff
@@ -278,6 +278,8 @@ class DbSlice {
   //! at a time of the call.
   uint64_t RegisterOnChange(ChangeCallback cb);
 
+  void CallChangeOnAllLessThanVersion(DbIndex db_ind, PrimeIterator it, uint64_t version);
```

Comment:
I think the name could be improved :)
`void ApplyChangesToPreviousSnapshots(..., uint64_t upper_bound_ver);`